pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -739,29 +746,36 @@ def build_row(options, name, svr):
                     svr.certfile, svr.keyfile, "\n".join(svr.mock_server)]
         return [name, svr.server, "\n".join(svr.mock_server)]
 
-    connections = context.connections_repo
+    connections_repo = context.connections_repo
     output_format = validate_output_format(context.output_format, 'TABLE')
 
     # build the table structure
     rows = []
     cur_sym = '*'  # single char representing current connection
     dflt_sym = '#'  # single char representing persisted default connection
 
-    if connections.file_exists():
-        for name, svr in connections.items():
-            cc = cur_sym if is_current_connection(context, svr) else ''
-            dc = dflt_sym if is_default_connection(context, svr) else ''
-            name = '{}{}{}'.format(cc, dc, name)
-            rows.append(build_row(options, name, svr))
+    try:
+        if connections_repo.file_exists():
+            for name, svr in connections_repo.items():
+                cc = cur_sym if is_current_connection(context, svr) else ''
+                dc = dflt_sym if is_default_connection(context, svr) else ''
+                name = '{}{}{}'.format(cc, dc, name)
+                rows.append(build_row(options, name, svr))
+    except Exception as exc:  # pylint: disable=broad-except
```

Comment:
The code in the try block cannot possibly raise any exception. Suggest to remove the exception handler.

Reply:
DONE